### ISSUE TITLE
Display unit price on cart page

### ISSUE THREAD

**Issue by sauloperez** [openfoodfoundation/openfoodnetwork#6498]
## Description

**- As a:** user
**- On page:** Cart sidebar and `/cart` page
**- I want to be able to do:** See the unit price of each of the line item's variants in the cart dialog and the order's line items table.

## Acceptance Criteria & Tests

The unit price should include any fees listed in the price breakdown like done in #6497.

Check the design specs linked in https://github.com/openfoodfoundation/openfoodnetwork/issues/6449#issue-756226197. Watch out mobile!

1. Fill up your cart with some products from a shopfront
2. Click on the top-right cart icon. The sidebar should show each line item's unit price.
3. Fill up the checkout form and place the order. The order confirmation should display each line item's unit price under the price.

The unit price should correctly match the price displayed for the variant in both the variant row and the bulk modal.

**Comment by jibees** [comment]
As I can see in the Figma file there's no need to display unit price into the checkout page:
<img width="703" alt="Capture d’écran 2021-02-16 à 23 34 53" src="https://user-images.githubusercontent.com/296452/108130255-a8e37a00-70af-11eb-9d24-4d45f295f1e8.png">

On the other hand, I guess it should be displayed in the `/cart` page.
<img width="838" alt="Capture d’écran 2021-02-16 à 23 35 43" src="https://user-images.githubusercontent.com/296452/108130322-c57fb200-70af-11eb-85f5-00880492a5da.png">

What do you think? Should I edit the issue description?

**Comment by Matt-Yorkley** [comment]
Yeah I think it's okay on the cart page, but we don't need it in the checkout :+1:

**Comment by jaycmb** [comment]
Yes, we display in on the cart page but not needed in the checkout. I ll update the description.

And just flagging that the checkout page is currently in re-design anyways.

**Comment by Matt-Yorkley** [comment]
Actually now that I think about it, does this makes sense on the cart page? The point of it is to be able to compare prices when selecting, but on the cart page there's nothing to compare with other than what's already been selected.

**Comment by jaycmb** [comment]
Yes we discussed this as well.
From a legal point it´s nowhere clearly stated where it is required and where not, from a user perspective it can still be useful to know the unit prices when having different products in the cart - but of course not as much as for comparison.
Also - to be fair - the whole feature would be more useful, if shoppers could compare across different shops in one cart, which they can´t on OFN.

In the end we oriented on how shopify does it, including unit prices on the cart pages:
https://flowdocs.wetheme.com/faqs/price-per-unit
